whois-api/src/main/java/net/ripe/db/whois/api/httpserver/RewriteEngine.java

```diff
@@ -74,4 +76,57 @@ public RewriteHandler getRewriteHandler() {
         return rewriteHandler;
     }
 
+    private void restRedirectRules(final VirtualHostRuleContainer virtualHost) {
+        virtualHost.addRule(new CaseInsensitiveRewriteRegexRule(
+    "^/(search|geolocation|metadata|abuse-contact|references|autocomplete|domain)/?(.*)$",
+    "/whois/$1/$2"
+        ));
+
+        virtualHost.addRule(
+            new HttpTransportRule("https",
+                new HttpMethodRule(Set.of("post", "put", "delete"), new CaseInsensitiveRewriteRegexRule(
```

Comment:
Use Jakarta or Jetty HttpMethod for constant values?

Reply:
Jetty :)